src/main/java/org/spongepowered/api/entity/living/player/tab/TabListEntrySet.java

```diff
@@ -24,81 +24,20 @@
  */
 package org.spongepowered.api.entity.living.player.tab;
 
-import net.kyori.adventure.text.Component;
-import org.checkerframework.checker.nullness.qual.Nullable;
+import org.spongepowered.api.data.persistence.DataSerializable;
 import org.spongepowered.api.entity.living.player.Player;
-import org.spongepowered.api.entity.living.player.server.ServerPlayer;
 
 import java.util.Collection;
 import java.util.Optional;
 import java.util.UUID;
 
 /**
- * Represents a {@link Player}'s tab list.
+ * Represents a {@link Player}'s tab list entries.
  */
-public interface TabList {
+public interface TabListEntrySet extends DataSerializable {
```

Comment:
Maybe just `TabListEntries`? I don't like "Set" in the name

Reply:
Yeah, that's fine! I really wasn't sure what to call it. I'll change the name when I'm home.